pygnore/__init__.py,sha256=IMjkMO3twhQzluVTo8Z6rE7Eg-9U79_LGKMcsWLKBkY,22
pygnore/api.py,sha256=c2Gxf5QpcRWSzYs2mYdpT4F5dapIEVhhfki1QHQ03cY,1269
pygnore/exceptions.py,sha256=KOQC8fRkSzOtvyqVgjD-hMp2VtAnZOCaXKeEq1tIYc8,47
pygnore-0.1.0.dist-info/LICENSE,sha256=ws2Y69N1VcSdSKVmlWD-_ltaK1P4MWFOV_SrkiNVa9A,1074
pygnore-0.1.0.dist-info/METADATA,sha256=XNOmQ-ZlRW0KcjNBe4HddGiIN1HA1kETNyXQSkFTT4s,1479
pygnore-0.1.0.dist-info/WHEEL,sha256=MYFsq5fFBwF_oyJgrOoFmYYB1K6Sw7MxY-0897ZLbdM,92
pygnore-0.1.0.dist-info/top_level.txt,sha256=sjxL9xe9YcDyfmrWq8ye3yWhL7_xBtwvBL4goyneI_8,8
pygnore-0.1.0.dist-info/RECORD,,
